x/upgrade/keeper/keeper_test.go

```diff
@@ -29,29 +31,34 @@ type KeeperTestSuite struct {
 	ctx           sdk.Context
 	msgSrvr       types.MsgServer
 	addrs         []sdk.AccAddress
+	encCfg        moduletestutil.TestEncodingConfig
 }
 
 func (s *KeeperTestSuite) SetupTest() {
-	var (
-		bankKeeper    bankkeeper.Keeper
-		stakingKeeper *stakingkeeper.Keeper
-	)
 
 	homeDir := filepath.Join(s.T().TempDir(), "x_upgrade_keeper_test")
-	appConfig := depinject.Configs(testutil.AppConfig, depinject.Supply(simtestutil.NewAppOptionsWithFlagHome(homeDir)))
-	app, err := simtestutil.Setup(appConfig, &s.upgradeKeeper, &bankKeeper, &stakingKeeper)
-	s.NoError(err)
-	s.upgradeKeeper.SetVersionSetter(app.BaseApp)
+	s.encCfg = moduletestutil.MakeTestEncodingConfig(upgrade.AppModuleBasic{})
+	key := sdk.NewKVStoreKey(types.StoreKey)
+	testCtx := testutil.DefaultContextWithDB(s.T(), key, sdk.NewTransientStoreKey("transient_test"))
+
+	s.baseApp = baseapp.NewBaseApp(
+		"nft",
```

Comment:
why the name "nft"?

Reply:
I followed nft mocks while writing mocks for upgrade and I forgot to change this, thanks for pointing @kocubinski